src/org/usfirst/frc2175/command/single/ShiftToHighGearCommand.java

```diff
@@ -0,0 +1,48 @@
+package org.usfirst.frc2175.command.single;
+
+import org.usfirst.frc2175.subsystem.RobotSubsystems;
+import org.usfirst.frc2175.subsystem.drivetrain.DrivetrainSubsystem;
+
+import edu.wpi.first.wpilibj.command.Command;
+
+/**
+ *
+ */
+public class ShiftToHighGearCommand extends Command {
+    private final DrivetrainSubsystem drivetrainSubsystem;
+
+    public ShiftToHighGearCommand(RobotSubsystems robotSubsystems) {
+        this.drivetrainSubsystem = robotSubsystems.getDrivetrainSubsystem();
+        requires(drivetrainSubsystem);
```

Comment:
Ok, so here's an interesting thing...because this command requires the drivetrain subsystem, it will actually interrupt the command to drive. In the past we've just removed the "requires" line so that doesn't happen.